doc/environ.md

```diff
@@ -108,6 +108,18 @@ DAOS debug logs (both server and client) are written to /tmp/daos.log by
 default. This can be modified by setting this environment variable
 ("D_LOG_FILE=/tmp/daos_server").
 
+### `D_LOG_SIZE`
+
+DAOS debug logs (both server and client) have a 1GB file size limit by
+default. When this limit is reached, the current log file is closed and
+renamed with a .old suffix, and a new one is opened. This mechanism
+will repeat each time the limit is reached, meaning that available
+saved log records could be found in both ${D_LOG_FILE} and last
+generation of ${D_LOG_FILE}.old files, to a maximum of the most recent
+2*D_LOG_SIZE records.
+This can be modified by setting this environment variable
+("D_LOG_SIZE=536870912").
```

Comment:
Should add a comment mentioning that human-readable sizes (i.e. 1G) can be used here?  Something like:

```suggestion
("D_LOG_SIZE=536870912").  Sizes can also be specified in human-readable form using `k`, `m`, `g`, `K`, `M`, and `G`.  The lower-case specifiers are base-10 multipliers and the upper case specifiers are base-2 multipliers.
```

Reply:
Ok, agree! BTW, I need to move this to doc/admin/env_variables.md as doc/environ.md has recently been removed